addon/components/lazy-render-wrapper.js

```diff
@@ -102,7 +103,7 @@ export default Ember.Component.extend({
 
 			return true;
 
-		} else if (this.get('isShown') || this.get('tooltipIsVisible')) {
+		} else if (!this.get('disabled') && (this.get('isShown') || this.get('tooltipIsVisible'))) {
 
 			this.set('_hasRendered', true);
 			return true;
```

Comment:
> "And what about in 128, if i skip the parent event stuff, and then disabled gets set to false, would that be rerun anywhere else?"

You're right. If we people change isDisabled we would need to add the event listeners to the $parent. I think maybe we should make the $parent event handling a function that can be called on didInsertElement and by observing `isDisabled`. 

We'd need good tests to cover this. It's getting fairly complex